config_app/js/core-config-setup/config-setup-tool.html

```diff
@@ -421,6 +421,132 @@
+                  <option value="elasticsearch">Elasticsearch</option>
+                  <option value="database">Database</option>
+                </select>
+              </td>
+          </table>
+
+          <table class="config-table" ng-if="mapped.LOGS_MODEL == 'elasticsearch'">
+            <tr>
+              <td><strong>Elasticsearch config</strong></td>
+            </tr>
+            <tr>
+              <td>Elasticsearch hostname:</td>
+              <td>
+                <span class="config-string-field" binding="mapped.LOGS_MODEL_CONFIG.elasticsearch_config.host" placeholder="The Elasticsearch server hostname"
+                      pattern="{{ HOSTNAME_REGEX }}" validator="validateHostname(value)"></span>
+              </td>
+            </tr>
+            <tr>
+              <td>Elasticsearch port:</td>
+              <td>
+                <span class="config-numeric-field" binding="mapped.LOGS_MODEL_CONFIG.elasticsearch_config.port" default-value="9200"></span>
+                <div class="help-text">
+                  Access to this port and hostname must be allowed from all hosts running
+                  the enterprise registry
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td>Elasticsearch access key:</td>
+              <td>
+                <span class="config-string-field" binding="mapped.LOGS_MODEL_CONFIG.elasticsearch_config.access_key"
+                      placeholder="The Elasticsearch access key"
+                </span>
+              </td>
+            </tr>
+            <tr>
+              <td>Elasticsearch secret key:</td>
+              <td>
+                <span class="config-password-field" binding="mapped.LOGS_MODEL_CONFIG.elasticsearch_config.secret_key"
+                      placeholder="The Elasticsearch secret key"
+                </span>
+              </td>
+            </tr>
+            <tr>
+              <td>AWS region:</td>
+              <td>
+                <span class="config-string-field" binding="mapped.LOGS_MODEL_CONFIG.elasticsearch_config.aws_region"
+                      placeholder="The AWS region (optional)" is-optional="true"
+                </span>
+              </td>
+            </tr>
+            <tr>
+              <td>Index prefix:</td>
+              <td>
+                <span class="config-string-field" binding="mapped.LOGS_MODEL_CONFIG.elasticsearch_config.index_prefix"
+                      default-value="logentry_" placeholder="The Elasticsearch index_prefix"
+                </span>
+              </td>
+            </tr>
+            <tr>
+              <td class="non-input">Logs Producer:</td>
+              <td>
+                <select class="form-control" ng-model="mapped.LOGS_MODEL_CONFIG.producer">
+                  <option value="kinesis_stream">Kinesis</option>
+                  <option value="elasticsearch">Elasticsearch</option>
+                </select>
```

Comment:
Make this default to "elasticsearch" in UI. It is blank to start with which then omits the `producer` entry in the config.yaml.

Reply:
Done